### PR TITLE
VPN documentation proofreading

### PR DESCRIPTION
**Pull request by whyrlpool**
## Change Summary

openconnect.rst: spelling errors corrected
dmvpn.rst: small spelling error corrected
ipsec.rst: updated justification, changes to phrasing
l2tp.rst: updated justification, changes to phrasing
sstp.rst: updated justification, changes to phrasing

## Related Task(s)

N/A, documentation change.

## Backport

Ideally back to at least 1.4, but likely applicable to 1.3 as well.

## Checklist:

- [x] I have read the [**CONTRIBUTING**](https://github.com/vyos/vyos-documentation/blob/current/CONTRIBUTING.md) document

### CODE REVIEW

File: docs/configuration/vpn/ipsec.rst [hunk 1]
```diff
@@ -13,10 +13,10 @@ address, which makes it easier to setup static routes or use dynamic routing
 protocols without having to modify IPsec policies. The other advantage is that
 it greatly simplifies router to router communication, which can be tricky with
 plain IPsec because the external outgoing address of the router usually doesn't
-match the IPsec policy of typical site-to-site setup and you need to add special
-configuration for it, or adjust the source address for outgoing traffic of your
-applications. GRE/IPsec has no such problem and is completely transparent for
-the applications.
+match the IPsec policy of a typical site-to-site setup and you would need to
+add special configuration for it, or adjust the source address for your
+applications' outgoing traffic. GRE/IPsec has no such problem and is completely
```
Comment (c-po) [inline docs/configuration/vpn/ipsec.rst]: Missplaced `'`?

**Review by c-po — CHANGES_REQUESTED** [review]

File: docs/configuration/vpn/ipsec.rst (hunk 1 above)
Comment (whyrlpool) [reply docs/configuration/vpn/ipsec.rst]: applications are plural, and it’s a possessive apostrophe
I can reword it to make it simpler if you’d prefer.

Comment (c-po) [reply docs/configuration/vpn/ipsec.rst]: Shouldn't it be `application's` then instead?

Comment (dmbaturin) [reply docs/configuration/vpn/ipsec.rst]: The intended syntax here is `possessive(plural('application'))`. In different words, "or adjust the source address for outgoing traffic of your applications". I might prefer the latter, but I have no objections against "applications' outgoing traffic".

Comment (whyrlpool) [reply docs/configuration/vpn/ipsec.rst]: Your wording here is clearer. I’ll update the wording to match, thanks!

**Comment by dmbaturin** [comment]
Thanks!
